pkg/images/images.go

```diff
@@ -39,8 +39,9 @@ type ResolverOpt func(name.Reference) (name.Reference, error)
 
 // Resolver provides functionality to resolve an RKE2 image name to a reference.
 type Resolver struct {
-	registry  name.Registry
-	overrides map[string]name.Reference
+	registry   name.Registry
+	repoPrefix string // optional path segment to prepend to image repository
```

Comment:
`repository` is a Docker Hub thing. OCI spec refers to this as the name, and that is what we should refer to it as as well. Ref:https://github.com/opencontainers/distribution-spec/blob/main/spec.md#endpoints


```suggestion
	namePrefix string // optional path segment to prepend to image name
```